lib/src/main/java/uk/gov/di/ipv/cri/passport/library/domain/NameParts.java

```diff
@@ -0,0 +1,82 @@
+package uk.gov.di.ipv.cri.passport.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import uk.gov.di.ipv.cri.passport.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({"value", "type", "validFrom", "validUntil"})
```

Comment:
Do we need to specify the json property order?

Reply:
No, don't think we do!